teuthology/config.py

```diff
@@ -1,7 +1,12 @@
 import os
 import yaml
 import logging
-import collections
+from sys import sys_version_info
+if sys_version_info.major >= 3 and sys_version_info.minor >= 10 and
+   sys_version_info.micro >= 4:
+    from collections.abc import MutableMapping
```

Comment:
instead of hardwiring the feature of a certain python standard library module to the python version, i'd suggest just `try`.

Reply:
Right! You mean -

```
try:
 from collections.abc import MutableMapping
except
 from collections import MutableMapping

```

Reply:
Right! You mean -

```
try:
    from collections.abc import MutableMapping
except:
    from collections import MutableMapping

```

Reply:
yeah, better off be more explicit:

```py
try:
 from collections.abc import MutableMapping
except ImportError:
 from collections import MutableMapping
```

Reply:
@tchaikov Right. I put it only roughly then.

Reply:
@tchaikov I've made the change. PTAL. And thanks again!